src/common/cli_helper.cpp

```diff
@@ -15,6 +15,9 @@ void makeAbsolutePaths(YAML::Node& config,
       return nodePath;
 
     // replace relative path w.r.t. config directory
+  #if WITHOUT_EXCEPTIONS
+      return canonical(filesystem::Path{nodePath}, configDir).string();
```

Comment:
So what happens if the file does not exist? No error handling?

Reply:
This comment is not valid anymore as I reverted this change 👍